search-app/docs/api.md

```diff
@@ -4661,6 +4664,8 @@ permission.
 
     curl -i -XPOST -H "Content-Type: application/json" %CMR-ENDPOINT%/clear-scroll -d '{ "scroll_id" : "xxxx"}'
 
+It returns HTTP status code 204 when successful.
```

Comment:
We may not be returning anything, but isn't the post a command to do something? Would 200 be better?

Reply:
204 is success without content.